composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/requirements.txt

```diff
@@ -0,0 +1,2 @@
+apache-airflow[gcp]==1.10.12
+apache-airflow-backport-providers-google==2020.11.13
```

Comment:
Will the plan be to update these versions manually whenever we have an upgrade?

Reply:
Already answered in your blog: 
"I specifically pin the operators to a specific version so it is always clear what version is installed in my environment. "

Reply:
In general also, we always have a bot watching our versions to upgrade them, so I never have to worry